apps/star_wars/mix.exs

```diff
@@ -0,0 +1,49 @@
+defmodule StarWars.Mixfile do
+  use Mix.Project
+
+  def project do
+    [app: :star_wars,
+     version: "0.0.1",
+     elixir: "~> 1.2",
+     elixirc_paths: elixirc_paths(Mix.env),
+     compilers: [:phoenix, :gettext] ++ Mix.compilers,
+     build_embedded: Mix.env == :prod,
+     start_permanent: Mix.env == :prod,
+     build_path: "../../_build",
+     config_path: "../../config/config.exs",
+     deps_path: "../../deps",
+     lockfile: "../../mix.lock",
+     aliases: aliases(),
+     deps: deps()]
+  end
+
+  # Configuration for the OTP application.
+  #
+  # Type `mix help compile.app` for more information.
+  def application do
+    [mod: {StarWars, []},
+     applications: [:absinthe_relay, :absinthe_plug]]
+  end
+
+  # Specifies which paths to compile per environment.
+  defp elixirc_paths(:test), do: ["lib", "web", "test/support"]
+  defp elixirc_paths(_),     do: ["lib", "web"]
+
+  # Specifies your project dependencies.
+  #
+  # Type `mix help deps` for examples and options.
+  defp deps do
```

Comment:
I think that for the illustrative purposes of this project, and to make it more maintainable as a _monolith_ we have to depend on the libraries of the parent app.

I think that all the common stuff should be in the top app and share as much as possible. For example, when we upgrade to absinthe 1.2.